### PR TITLE
typescript refactoring

### PR DESCRIPTION
**Pull request by rscotten**

### CODE REVIEW

File: src/database/helpers/index.ts [hunk 1]
```diff
@@ -9,22 +9,23 @@ export const getRandomInt = (min: number, max: number) =>
...
-    array.push(factory());
+export const fillArray = (
+  length: number,
+  factory: Function,
+  data: any[] = []
+): any[] => {
+  if (length) {
+    data.push(factory());
+    return fillArray(length - 1, factory, data);
   }
```
Comment (wscotten) [inline src/database/helpers/index.ts]: nice

File: src/shared/types/db/Member.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export interface Member {
+  userId: string;
+  role: string; // owner, admin, viewer
+  readonly invitedAt: number;
+}
```
Comment (wscotten) [inline src/shared/types/db/Member.ts]: Do we want to have the interfaces that compose the collections to be in the same folder as the collections themselves?

Comment (wscotten) [reply src/shared/types/db/Member.ts]: I dont know if this is a new collection but I dont think it is.

Comment (rscotten) [reply src/shared/types/db/Member.ts]: good question.  I made a "shared" folder for shared types that are not collections.  I also made the "types" folder a root directory.

**Pull request merged**